src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -1379,12 +1379,28 @@ private void CheckModelAndSyntaxNodeToSpeculate(CSharpSyntaxNode syntax)
                 Debug.Assert((object)container == null);
                 TypeSymbol containingType = binder.ContainingType;
                 TypeSymbol baseType;
-                if ((object)containingType == null || (object)(baseType = containingType.BaseTypeNoUseSiteDiagnostics) == null)
+                if ((object)containingType == null)
                 {
                     throw new ArgumentException(
                         "Not a valid position for a call to LookupBaseMembers (must be in a type with a base type)",
```

Comment:
I'm probably missing something obvious, but shouldn't these messages be localized?  It might also be nice to use `nameof`.